udata/forms/fields.py

```diff
@@ -541,6 +541,8 @@ def _value(self):
 
     def process_formdata(self, valuelist):
         if not valuelist:
+            if valuelist == []:
```

Comment:
:warning:  valuelist is also `[]` if the field is simply missing! Thus, we're emptying `datasets` if the value is not set. I'm confused about how we're supposed to differentiate between an empty list and a missing value here?

We may de-prioritize this ticket if we go for https://github.com/ecolabdata/ecospheres-front/issues/52.
But I guess this issue is probably not encountered on Topics only?